rtl/eclrtl/rtlrecord.hpp

```diff
@@ -127,7 +127,7 @@ class ECLRTL_API RtlRecordSize : public IRecordSize, public RtlCInterface
         return offsetCalculator.getRecordSize();
     }
 
-    virtual size32_t getFixedSize()
+    virtual size32_t getFixedSize() const
```

Comment:
Yes, that code is only is 7.0 - and is currently unused (which is why the problem wasn't discovered)